assets/prometheus-adapter/deployment.yaml

```diff
@@ -33,14 +33,18 @@ spec:
     spec:
       affinity:
         podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchLabels:
-                app.kubernetes.io/component: metrics-adapter
-                app.kubernetes.io/managed-by: cluster-monitoring-operator
-                app.kubernetes.io/name: prometheus-adapter
-                app.kubernetes.io/part-of: openshift-monitoring
-            topologyKey: kubernetes.io/hostname
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/component: metrics-adapter
+                  app.kubernetes.io/managed-by: cluster-monitoring-operator
```

Comment:
This label will be removed after https://github.com/openshift/cluster-monitoring-operator/pull/1138. Seems like that PR is blocking this one and we will need to rebase :(